pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -88,7 +99,39 @@ func (v *vpnShoot) WaitCleanup(ctx context.Context) error {
 }
 
 func (v *vpnShoot) computeResourcesData() (map[string][]byte, error) {
-	registry := managedresources.NewRegistry(kubernetes.ShootScheme, kubernetes.ShootCodec, kubernetes.ShootSerializer)
-
-	return registry.AddAllAndSerialize()
+	var (
+		registry = managedresources.NewRegistry(kubernetes.ShootScheme, kubernetes.ShootCodec, kubernetes.ShootSerializer)
+
+		serviceAccount = &corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "vpn-shoot",
+				Namespace: metav1.NamespaceSystem,
+				Labels:    map[string]string{LabelKey: LabelValue},
+			},
+		}
+
+		service = &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "vpn-shoot",
+				Namespace: metav1.NamespaceSystem,
+				Labels:    map[string]string{LabelKey: LabelValue},
+			},
+			Spec: corev1.ServiceSpec{
+				Selector: map[string]string{LabelKey: LabelValue},
+				Type:     corev1.ServiceTypeLoadBalancer,
+				Ports: []corev1.ServicePort{
+					{
+						Name:       "openvpn",
+						Port:       servicePort,
+						TargetPort: intstr.FromInt(int(containerPort)),
+						Protocol:   corev1.ProtocolTCP,
+					},
+				},
+			},
+		}
+	)
+	return registry.AddAllAndSerialize(
+		serviceAccount,
+		service,
```

Comment:
See above: You only want to add this `service` if `ReversedVPN` is `false` (adapt the unit test accordingly for both cases)